torchao/dtypes/uintx/cutlass_int4_packed_layout.py

```diff
@@ -110,8 +108,7 @@ def from_plain(
         _layout: Layout,
     ):
         assert zero_point is None or torch.all(zero_point == 0)
-
-        int_data_s4 = ((int_data[:, 1::2] & 0xF) << 4) | (int_data[:, 0::2] & 0xF)
+        int_data_s4 = ((int_data[..., 1::2] & 0xF) << 4) | (int_data[..., 0::2] & 0xF)
```

Comment:
just curious here, what's the difference between `...` and `:`?

Reply:
Need to look into 3D and higher-dimensional tensors to spot the difference: for example, for 3D tensor `x`, it holds that `x[..., -1]` is `x[:, :, -1]`, while `x[:, -1]` is `x[:, -1, :]`.